docs/core/testing/unit-testing-with-mstest.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Use MSTest with .NET Core
+description: How to use MSTest with .NET Core
+keywords: MSTest, .NET, .NET Core
+author: ncarandini
+ms.author: wiwagn
+ms.date: 02/10/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: ed447641-3e85-4e50-b7ed-004630048a3e
+---
+
+# Unit testing with MSTest and .NET Core
+
```

Comment:
should we add a warning on top of the unit testing topics to say to which version the topic applies to? similar to what've done with the topics in the preview3 folder? Because this one applies to RC4 and the other one to preview 2